### PR TITLE
feat(documentation): migrate post-icon to storybook v7 + lit

### PR DESCRIPTION
**Pull request by imagoiq**
Some changes:
* Title is auto-generated now, so I removed it
* There is a double line-break between components in the source view. We can remove it, but I thought that visual it's not bad to make a larger separation.

|  Before |  After |
|---|---|
| image |  image |

* I removed the following parameters as it didn't seem to have an impact on anything:
```
  parameters: {
    controls: {
      exclude: ['Base', 'Flip Horizontally', 'Flip Vertically', 'Scale', 'Rotate', 'Animation'],
    },
  },
 ```

### CODE REVIEW

File: packages/documentation-v7/src/stories/icons/components/icon.stories.ts [hunk 1]
```diff
@@ -0,0 +1,223 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import { spread } from '@open-wc/lit-helpers';
```
Comment (imagoiq) [inline packages/documentation-v7/src/stories/icons/components/icon.stories.ts:2]: For more info about this import: https://github.com/lit/lit/issues/923

File: packages/documentation-v7/.storybook/preview.ts [hunk 2]
```diff
@@ -7,6 +10,26 @@ import './helpers/register-web-components';
...
+  jsxSingleQuote: false,
+  trailingComma: 'es5',
+  bracketSpacing: true,
+  bracketSameLine: false,
+  arrowParens: 'always',
+  htmlWhitespaceSensitivity: 'css',
+  endOfLine: 'lf',
+  embeddedLanguageFormatting: 'off',
+  singleAttributePerLine: false,
+};
```
Comment (imagoiq) [inline packages/documentation-v7/.storybook/preview.ts:31]: No change

File: packages/documentation-v7/src/stories/icons/components/icon.docs.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import LinkTo from '@storybook/addon-links/react';
+import { Meta, Canvas, Controls, Story, Source, ArgsTable } from '@storybook/blocks';
+
+import * as IconStories from './icon.stories';
+
+<Meta of={IconStories} />
+
+# Icon
+
+<p class="lead">
```
Comment (alizedebray) [inline packages/documentation-v7/src/stories/icons/components/icon.docs.mdx:10]:
```suggestion
<div class="lead">
```

MDX2 is already rendering texts within `<p>` tags. Using a `<div>` here will prevent having nested paragraphs (and get rid of a console error in the meantime!)

File: packages/documentation-v7/src/stories/icons/components/icon.stories.ts [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+import { Args, Meta, StoryObj } from '@storybook/web-components';
+import { Components } from '@swisspost/design-system-components';
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import docsPage from './icon.docs.mdx';
+
+const meta: Meta = {
+  component: 'post-icon',
+  parameters: {
+    docs: {
```
Comment (alizedebray) [inline packages/documentation-v7/src/stories/icons/components/icon.stories.ts:12]: I suppose this parameter no longer apply with v7.

File: packages/documentation-v7/src/stories/icons/components/icon.docs.mdx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+import * as IconStories from './icon.stories';
+
+<Meta of={IconStories} />
+
+# Icon
+
+<div class="lead">
+  Our <code>&lt;post-icon&gt;</code> component renders SVGs, so it scales quickly and easily and can
+  be styled with CSS.
+</p>
```
Comment (imagoiq) [inline packages/documentation-v7/src/stories/icons/components/icon.docs.mdx:13]:
```suggestion
</div>
```

**Review by oliverschuerch — COMMENTED** [review]
We need to install and configurate the addon @pxtrn/storybook-addon-docs-stencil to make the story argTable work as expected!

**Comment by imagoiq** [comment]
@oliverschuerch is it ready to merge from your point of view?

**Pull request merged**